test/infra/main.tf

```diff
@@ -56,6 +57,12 @@ variable "lambda_env" {
   default     = {}
 }
 
+variable "tags" {
```

Comment:
Do we need this?
I believe is already declared in variables.tf

Reply:
If I delete this I got an error from tests: 
`Error: module 'lambda': unknown variable referenced: 'tags'; define it with a 'variable' block`